### PR TITLE
Fix error message caching

### PR DESCRIPTION
**Pull request by lab900-virginie-slagmolen**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -60,7 +60,8 @@
     "nouislider": "^14.7.0",
     "rxjs": "~6.6.7",
     "tslib": "^2.3.1",
-    "zone.js": "~0.11.4"
+    "zone.js": "~0.11.4",
+    "memo-decorator": "^2.0.1"
```
Comment (lab900-gino-vanhoudt) [inline package.json:64]: Why is this added? it's not used anywhere?

**Review by lab900-gino-vanhoudt — CHANGES_REQUESTED** [review]

**Review by lab900-gino-vanhoudt — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (lab900-winand-vandenbergh) [reply package.json:64]: It's a peer dependency for the ui library. This caused the showcase to fail on startup, no idea why this wasn't added earlier.
